### PR TITLE
Update translation.json for French translation 

### PR DESCRIPTION
**Pull request by Ekaekale**
Solves: #8

### CODE REVIEW

File: locales/fr/translation.json [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
-	"PM": "PM",
-	"Clear All Data!": "Clear All Data!",
+	"Mon Tue Wed Thu Fri Sat": "Lun Mar Mer Jeu Ven Sam",
+	"AM": "Matin",
+	"PM": "APRÈS-MIDI",
+	"Clear All Data!": "Remettre À Zéro!",
 	"Usage": "Usage",
-	"buyPriceInfo": "- The <1>Buy Price</1> value is for your own island. It doesn't matter if you buy turnips on another island!",
-	"priceChangeInfo": "- Prices change <1>twice a day.</1> Be sure to log them. (We save your data in your device).",
+	"buyPriceInfo": "- Le <1>Buy Price</1> est celui de ta propre île. Ce n'a pas d'importance si tu as acheté des navets sur une autre île!",
```
Comment (mtaylor76) [inline locales/fr/translation.json:14]: Buy Price within the <1> tags looks like it was left untranslated

File: locales/fr/translation.json [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
+	"priceChangeInfo": "- Les prix changent <1>deux fois par jour.</1> N'oubliez pas de vérifier. (Nous sauvegardons vos données dans votre appareil).",
 	"Changelog": "Changelog",
```
Comment (mtaylor76) [inline locales/fr/translation.json:16]: Was "Changelog" left untranslated, or is the French word the same? Not sure.

File: locales/fr/translation.json (hunk 1 above)
Comment (Ekaekale) [reply locales/fr/translation.json:14]: Oops. I forgot to translate it, my bad.

File: locales/fr/translation.json (hunk 2 above)
Comment (Ekaekale) [reply locales/fr/translation.json:16]: We use the same word in French. =) (Anglicism)

**Comment by elxris** [comment]
I'm adding a new feature. Could we wait a little for this?

**Comment by elxris** [comment]
I have pushed the new feature. We can continue. Thank you for your patience!

**Comment by elxris** [comment]
Please review 1.3 changes, and I have added "Guaranteed Min", "guaranteedMinInfo" and "changelog1.4"

**Comment by elxris** [comment]
I will push 1.4 with this translation missing. Just to deliver some other features! But as soon as this is ready I will deploy them. Thank you! ❤️

**Comment by elxris** [comment]
I'm adding a Work in Progress label. Please remove it when it's done! Thank you!

File: locales/fr/translation.json [hunk 3]
```diff
@@ -1,28 +1,28 @@
...
   "guaranteedMinInfo": "- The <1>Guaranteed Min</1> value means that you will see at least this price at some point of the week.",
-  "Changelog": "Changelog",
-  "changelog1.1": "1.1: Clear data button! Please use it responsibly!",
-  "changelog1.2a": "1.2a: Fixed ranges of patterns",
-  "changelog1.3": "1.3: Animations fixed. New icon (<1>@mtaylor76</1>). New horizontal marker for buy price (<3>@pudquick</3>)!",
+	"Changelog": "Changelog",
+	"changelog1.1": "1.1: Bouton de RàZ! Attention en l'utilisant !",
+	"changelog1.2a": "1.2a: Fix des patterns",
+	"changelog1.3": "1.3: Animations fixed. New icon (<1>@mtaylor76</1>). New horizontal marker for buy price (<3>@pudquick</3>)!",
   "changelog1.4": "1.4: Guaranteed Min. (<1>@capoferro & @nanoNago</1>). Localization Support (<3>@mtaylor76</3>). French Localization (<5>@Ekaekale</5>). New Currency Icon (<7>@alyphen</7>). Bug Reporters (<9>@FoxFireX</9>).",
```
Comment (Noelierx) [inline locales/fr/translation.json:22]:
```suggestion
  "changelog1.4": "1.4: Minimum Garanti. (<1>@capoferro & @nanoNago</1>). Support Traduction (<3>@mtaylor76</3>). Traduction Française (<5>@Ekaekale</5>). Nouvelle icône de monnaie (<7>@alyphen</7>). Rapporteurs de bugs (<9>@FoxFireX</9>).",

File: locales/fr/translation.json [hunk 4]
```diff
@@ -1,28 +1,28 @@
...
+	"priceChangeInfo": "- Les prix changent <1>deux fois par jour.</1> N'oubliez pas de vérifier. (Nous sauvegardons vos données dans votre appareil).",
   "guaranteedMinInfo": "- The <1>Guaranteed Min</1> value means that you will see at least this price at some point of the week.",
-  "Changelog": "Changelog",
-  "changelog1.1": "1.1: Clear data button! Please use it responsibly!",
-  "changelog1.2a": "1.2a: Fixed ranges of patterns",
-  "changelog1.3": "1.3: Animations fixed. New icon (<1>@mtaylor76</1>). New horizontal marker for buy price (<3>@pudquick</3>)!",
+	"Changelog": "Changelog",
+	"changelog1.1": "1.1: Bouton de RàZ! Attention en l'utilisant !",
+	"changelog1.2a": "1.2a: Fix des patterns",
+	"changelog1.3": "1.3: Animations fixed. New icon (<1>@mtaylor76</1>). New horizontal marker for buy price (<3>@pudquick</3>)!",
```
Comment (Noelierx) [inline locales/fr/translation.json:21]:
```suggestion
	"changelog1.3": "1.3: Animations fixées. Nouvelle icône (<1>@mtaylor76</1>). Nouveau marqueur horizontal pour le prix d'achat (<3>@pudquick</3>)!",
```

**Comment by Ekaekale** [comment]
@Noelierx Je ne suis pas très dégourdi avec l'utilisation de Github, j'espère avoir fait ce qu'il fallait pour ajouter vos traductions aux miennes ! Si ce n'est pas le cas, pouvez-vous m'aider afin de résoudre cela le plus rapidement possible ? Merci ! :)

**Comment by elxris** [comment]
Thank you! #17

**Pull request merged**